src/SelfServe/SelfServe.tsx

```diff
@@ -38,20 +38,26 @@ const loadTranslations = async (className: string): Promise<void> => {
 };
 
 const getDescriptor = async (selfServeType: SelfServeType): Promise<SelfServeDescriptor> => {
+  let descriptor: SelfServeDescriptor;
```

Comment:
You don't need this variable. You can always return directly from inside the switch statement:

`return selfServeExample.toSelfServeDescriptor();`